File: ILT.IHR.DTO/AssignmentRate.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace ILT.IHR.DTO
{
    public partial class AssignmentRate : AbstractDataObject
    {
        public int AssignmentRateID
        {
            get { return base.RecordID; }
            set { base.RecordID = value; }
        }
        public int AssignmentID { get; set; }
        [Required]
        [RegularExpression(@"^[1-9]\d*(\.\d+)?$")]
        public string BillingRate { get; set; }
        [Required]
        [RegularExpression(@"^[1-9]\d*(\.\d+)?$")]
        public string PaymentRate { get; set; }
        public DateTime StartDate { get; set; }
        public DateTime? EndDate { get; set; }
        public Boolean IsFLSAExempt { get; set; }
    }
}

File: ILT.IHR.APP/ClientApp/src/app/core/interfaces/AssignmentRate.ts
export class IAssignmentRate {
    AssignmentRateID: number;
    AssignmentID: number;
    BillingRate: string;
    PaymentRate: string;
    IsFLSAExempt: boolean;
    StartDate: Date;
    EndDate: Date;
    CreatedBy: string;
    CreatedDate: Date;
    ModifiedBy: string;
    ModifiedDate: Date;
    TimeStamp: ['']
}
export class IAssignmentRateDisplay {
    assignmentRateID: number;
    assignmentID: number;
    billingRate: string;
    paymentRate: string;
    isFLSAExempt: boolean;
    startDate: string;
    endDate: string;
    createdBy: string;
    createdDate: Date;
    modifiedBy: string;
    modifiedDate: Date;
}
